client/components/domains/domain-suggestion/test/index.js

```diff
@@ -27,16 +30,44 @@ describe( 'Domain Suggestion', function() {
 	} );
 
 	describe( 'has attributes', () => {
-		it( 'should have data-e2e-domain attribute for e2e testing', () => {
-			const domainSuggestion = shallow( <DomainSuggestion
-				domain="example.com" isAdded={ false }/> );
-			if ( domainSuggestion.props()[ 'data-e2e-domain' ] ) {
-				expect( domainSuggestion.props()[ 'data-e2e-domain' ] ).to.equal( 'example.com' );
-			} else {
+		context( 'AB test clickableButton', () => {
```

Comment:
Didn't even know contexts were an alias in mocha. These are pretty common in rspec, but I think I usually see nested describes for js. Do folks have any thoughts on what we like better?